### PR TITLE
HTTAT-22 - Added Mock Unit Tests

### PR DESCRIPTION
**Pull request by meghannfh**
# Description
Added Mock Unit tests and re-ran the tests.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
- [x] Unit test

# Checklist:
- [x] I have pulled code from main and re-tested changes before opening the PR
- [x] No new bugs or vulnerabilities have been found in the code
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by bob-fornal** [comment]
[HTTAT-22 - Mocks for Unit Test Cleanup](https://trello.com/c/kfCzIoKc/26-httat-22-mocks-for-unit-test-cleanup)

**Comment by meghannfh** [comment]
@bob-fornal is it best to resolve the conflicts before getting reviews?

**Comment by bob-fornal** [comment]
> @bob-fornal is it best to resolve the conflicts before getting reviews?

Generally, yes.

File: frontend/src/app/pages/home/home.component.spec.ts [hunk 1]
```diff
@@ -11,7 +12,10 @@ describe('HomeComponent', () => {
   beforeEach(async () => {
     await TestBed.configureTestingModule({
       imports: [HttpClientTestingModule],
-      declarations: [ AssistanceSelectionComponent, GradeLevelSelectionComponent, HomeComponent ]
+      declarations: [
+        HomeComponent,
```
Comment (bob-fornal) [inline frontend/src/app/pages/home/home.component.spec.ts:16]: OK. You're merge dropped the `AssistanceSelectionComponent` from the declarations. You will actually need to create ANOTHER mock, import it ... removing the existing `AssistanceSelectionComponent` import on line 3. Then add the new mock to the declarations.

**Review by bob-fornal — CHANGES_REQUESTED** [review]

File: frontend/src/app/pages/home/home.component.spec.ts (hunk 1 above)
Comment (bob-fornal) [reply frontend/src/app/pages/home/home.component.spec.ts:16]: Don't forget to run the Unit Test (`ng test`). As is, there should be an error while the tests pass.

Comment (bob-fornal) [reply frontend/src/app/pages/home/home.component.spec.ts:16]: You're mocking the `AssistanceSelectionComponent`.

Comment (meghannfh) [reply frontend/src/app/pages/home/home.component.spec.ts:16]: Alrighty!

**Review by bob-fornal — APPROVED** [review]

**Review by GUIMess — APPROVED** [review]

**Comment by bellsofaba** [comment]
Kudos Meg!

**Comment by CloudCoded** [comment]
Wow

**Pull request merged**